01_setup_secret_and_external_access.sql

```diff
@@ -6,17 +14,53 @@ CREATE OR REPLACE SECRET aws_glue_creds_secret_key
   PASSWORD = '';
 
 
--- create network rule to allow access to glue
+-- create network rule to allow access to glue, and athena
 -- note this is hardcoded to us-west-2 for now, but can be changed to whatever region necessary
 CREATE OR REPLACE NETWORK RULE aws_glue_access_rule
   MODE = EGRESS
   TYPE = HOST_PORT
   VALUE_LIST = ('glue.us-west-2.amazonaws.com', 
-                'glue.us-west-2.api.aws'); 
+                'glue.us-west-2.api.aws',
+                'athena.us-west-2.amazonaws.com',
+                'athena.us-west-2.api.aws'); 
 
 
 -- create external access integration to allow access to glue
 CREATE OR REPLACE EXTERNAL ACCESS INTEGRATION aws_glue_access_int
   ALLOWED_NETWORK_RULES = (aws_glue_access_rule)
   ALLOWED_AUTHENTICATION_SECRETS =(aws_glue_creds_secret_key)
+  ENABLED = true;
+
+
+-------------------------------------------------------------------
+-- OPTION 2: Create a security integration to allow access to AWS Glue using an AWS application ID
+-- additional notes on usign aws application id access
+-- create securty integration to allow access to aws application id
+
+
+CREATE OR REPLACE SECURITY INTEGRATION aws_glue_security_integration
+  TYPE = API_AUTHENTICATION
+  AUTH_TYPE = AWS_IAM
+  ENABLED = TRUE
+  AWS_ROLE_ARN = 'xxx';
```

Comment:
can you add a docs link where we discuss what permissions are needed for this role?